test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -664,8 +664,8 @@ def test_paste_mask_in_image(self):
     @skipIfUnsupportedMinOpsetVersion(11)
     @disableScriptTest()
     def test_mask_rcnn(self):
-        model = torchvision.models.detection.mask_rcnn.maskrcnn_resnet50_fpn(pretrained=False, min_size=200,
-                                                                             max_size=300)
+        model = torchvision.models.detection.mask_rcnn.maskrcnn_resnet50_fpn(pretrained=False, pretrained_backbone=True,
```

Comment:
why is `pretrained_backbone` True for `test_mask_rcnn` and `test_faster_rcnn`, but not for `test_keypoint_rcnn`?